cfme/utils/dockerbot/pytestbase/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM fedora:27
+FROM fedora:29
```

Comment:
30 ?

Reply:
3.6.8 default mapped

Reply:
You're right, should make that jump now.